vaadin-iron-list-flow-parent/vaadin-iron-list-flow-integration-tests/src/test/java/com/vaadin/flow/component/ironlist/tests/IronListViewIT.java

```diff
@@ -123,7 +123,7 @@ public void disabledListWithTemplates() throws InterruptedException {
 
     @Test
     public void rankedListWithEventHandling() {
-        validateListSize(findElement(By.id("using-events-with-templates")), 29);
+        validateListSize(findElement(By.id("using-events-with-templates")), 20);
```

Comment:
I guess this effectively disables virtualization, which might be missing the point for a virtual list component. I'd suggest to disable the test instead and open a ticket to properly fix it.